src/experimental/constructs/error-budget-alarm.test.ts

```diff
@@ -16,6 +20,66 @@ describe("The ErrorBudgetAlarmExperimental construct", () => {
     expect(Template.fromStack(stack).toJSON()).toMatchSnapshot();
   });
 
+  it("should create the correct resources for a simple Availability SLO", () => {
+    const stack = simpleGuStackForTesting();
+    const ec2App = new GuEc2App(stack, {
+      applicationPort: 3000,
+      app: "test-gu-ec2-app",
+      access: { scope: AccessScope.PUBLIC },
+      instanceType: InstanceType.of(InstanceClass.T4G, InstanceSize.MEDIUM),
+      monitoringConfiguration: { noMonitoring: true },
+      userData: "#!/bin/dev foobarbaz",
+      certificateProps: {
+        domainName: "code-guardian.com",
+      },
+      scaling: {
+        minimumInstances: 1,
+      },
+    });
+    new GuErrorBudgetAlarmExperimental(stack, {
+      sloName: "MapiFrontsAvailability",
+      sloTarget: 0.999,
+      badEvents: new MathExpression({
+        expression: "loadBalancer5xxErrors + instance5xxErrors",
+        usingMetrics: {
+          loadBalancer5xxErrors: ec2App.loadBalancer.metricHttpCodeTarget(HttpCodeTarget.TARGET_5XX_COUNT),
+          instance5xxErrors: ec2App.loadBalancer.metricHttpCodeElb(HttpCodeElb.ELB_5XX_COUNT),
+        },
+      }),
+      validEvents: ec2App.loadBalancer.metricRequestCount(),
+      snsTopicNameForAlerts: "test-sns-topic",
+    });
+    expect(Template.fromStack(stack).toJSON()).toMatchSnapshot();
+  });
+
+  it("should create the correct resources for a simple Latency SLO", () => {
+    const stack = simpleGuStackForTesting();
+    const ec2App = new GuEc2App(stack, {
+      applicationPort: 3000,
+      app: "test-gu-ec2-app",
+      access: { scope: AccessScope.PUBLIC },
+      instanceType: InstanceType.of(InstanceClass.T4G, InstanceSize.MEDIUM),
+      monitoringConfiguration: { noMonitoring: true },
+      userData: "#!/bin/dev foobarbaz",
+      certificateProps: {
+        domainName: "code-guardian.com",
+      },
+      scaling: {
+        minimumInstances: 1,
+      },
+    });
+    new GuErrorBudgetAlarmExperimental(stack, {
+      sloName: "MapiFrontsLatency",
```

Comment:
We might want to provide a nicer interface for users here. For example, the interface could ask users to input the `Duration` for a good event (i.e. should it be 200ms, 300ms, 1000ms etc.) and the construct could write out the statistic part for them